.github/workflows/labeler.yml

```
@@ -6,7 +6,13 @@ jobs:
   triage:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/labeler@main
+    - name: Add Labels to PR
+      uses: actions/labeler@main
       with:
-        repo-token: ${{ secrets.GITHUB_TOKEN }}
-        sync-labels: true
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          sync-labels: true
+    - name: Add Size Label to PR
+      uses: WizardOhio24/label-pr-size-action@master
+      with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          size-label-color: '10, very small, ffaf65 | 50, small, ff9e58 | 80, medium, ff9654 | 200, large, ff905d | 400, massive, ff905d'
```

```suggestion
          size-label-color: '10, very small, ffaf65 | 50, small, ff9e58 | 200, medium, ff9654 | 400, large, ff905d | 100000, massive, ff905d'
```
What would you think of this adjustment to the numbers?  I'm remembering a [study](https://smartbear.com/learn/code-review/best-practices-for-peer-code-review/) that found that code reviews should ideally cover ≲200–400 lines of code.  This change would make "large" PRs in the range of 200–400, and increase the upper limit for massive PRs.